src/message_interface/response.rs

```diff
@@ -56,3 +58,30 @@ pub enum Response {
     /// All went fine.
     Ok(()),
 }
+
+// Custom Debug implementation to not log secrets
+impl Debug for Response {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result {
+        match self {
+            Response::Account(account) => write!(f, "Account({:?})", account),
+            Response::Accounts(accounts) => write!(f, "Accounts({:?})", accounts),
+            Response::Addresses(addresses) => write!(f, "Addresses({:?})", addresses),
+            Response::AddressesWithUnspentOutputs(addresses) => {
+                write!(f, "AddressesWithUnspentOutputs({:?})", addresses)
+            }
+            Response::OutputIds(output_ids) => write!(f, "OutputIds({:?})", output_ids),
+            Response::Output(output) => write!(f, "OutputData({:?})", output),
```

Comment:
Out of curiosity, why `OutputData` ?

Reply:
Because it's not just an `Output`, it's also has metadata added
If you have a better name for that I'm happy to change

Reply:
Yesterday I didn't really understand what you meant, makes more sense to have `Output` there unless the enum variant also gets named `OutputData`